smithy-typescript-codegen/src/test/java/software/amazon/smithy/typescript/codegen/StructureGeneratorTest.java

```diff
@@ -53,22 +138,27 @@ public void testErrorStructureCodegen(String file, String expectedType) {
                 .model(model)
                 .fileManifest(manifest)
                 .settings(Node.objectNodeBuilder()
-                                  .withMember("service", Node.from("smithy.example#Example"))
-                                  .withMember("package", Node.from("example"))
-                                  .withMember("packageVersion", Node.from("1.0.0"))
-                                  .build())
+                        .withMember("service", Node.from("smithy.example#Example"))
+                        .withMember("package", Node.from("example"))
+                        .withMember("packageVersion", Node.from("1.0.0"))
+                        .build())
                 .build();
 
         new TypeScriptCodegenPlugin().execute(context);
         String contents = manifest.getFileString("/models/index.ts").get();
 
+        assertThat(contents, containsString(expectedType));
+        return contents;
+    }
+
+    private void testErrorStructureCodegen(String file, String expectedType) {
+        String contents = testStructureCodegen(file, expectedType);
+
         assertThat(contents, containsString("as __isa"));
         assertThat(contents, containsString("as __SmithyException"));
-        assertThat(contents, containsString(expectedType));
-        assertThat(contents, containsString("namespace Err {\n"
-                                            + "  export const isa = (o: any): o is Err => "
-                                            + "__isa(o, \"Err\");\n"
-                                            + "}"));
+        assertThat(contents, containsString("namespace Err {"));
+        assertThat(contents, containsString("  export const isa = (o: any): o is Err => "
+                                            + "__isa(o, \"Err\");\n"));
```

Comment:
The closing `}` was removed from this assertion.

Reply:
This test now just checks for isa just like the other test for NonErrorStructures https://github.com/awslabs/smithy-typescript/blob/2dccabd4e34e9245b041fdd7ddf84b342a9dc8ca/smithy-typescript-codegen/src/test/java/software/amazon/smithy/typescript/codegen/StructureGeneratorTest.java#L89-L91

It just checks for namespace name, and `isa` method. This is because the namespace now contains method `filterSensitiveLog` whose tests are added other functions.